src/js/tabs/pid_tuning.js

```diff
@@ -451,23 +451,27 @@ TABS.pid_tuning.initialize = function (callback) {
                     rpmFilterHarmonics_e.val(FILTER_DEFAULT.gyro_rpm_notch_harmonics);
                 }
 
-                if (checked !== (FC.FILTER_CONFIG.gyro_rpm_notch_harmonics !== 0)) { // if rpmFilterEnabled is not the same value as saved in the fc
+                const dialogSettings = {
```

Comment:
Is there a way for Agree / Cancel buttons to be on the right side of the dialog box?

Reply:
Yes, but it is using standarized dialog added in presets tab, you can try to pick a preset, is the same dialog.

Reply:
yeah, that probably should be a separate PR if we want to change how this dialog looks.